waves/src/WalletInfo.tsx

```diff
@@ -133,15 +140,20 @@ export default function WalletInfo({ balance }: WalletInfoProps) {
                                         </Button>
                                     </HStack>
                                 </VStack>
+                                <VStack bg="gray.100" align="center" borderRadius={"md"} p={1}>
+                                    <Button
+                                        size="md"
+                                        variant="wallet_button"
+                                        onClick={fundWallet}
+                                    >
+                                        Fund wallet
+                                    </Button>
```

Comment:
Can we have this button in development mode only please? See here for how to do this: https://create-react-app.dev/docs/adding-custom-environment-variables/